code/modules/maps/tether/levels/tether_turfs.dm

```diff
@@ -128,6 +129,11 @@ VIRGO3B_TURF_CREATE(/turf/simulated/mineral/floor)
 		mineral = GLOB.ore_data[mineral_name]
 		UpdateMineral()
 
+
+
+/turf/simulated/mineral/rich/indoors
+	outdoors = FALSE
```

Comment:
Especially when you made indoor variants?

Reply:
Probably not needed as well do to the aforementioned change above in removing the definition of mineral turfs being outdoors

Reply:
Am going to remove the outdoor definition for the proc above as well. The proc by default uses the turf's original outdoor settings when generating caves and such. By defining outdoor here it overrides it (in most cases)